docs/core/integration-quality-scale/rules/iqs011.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "IQS011 - Entities have a unique ID"
+---
+
+IQS011 is as follows:
+
+Entities have a unique ID
+
+## Reasoning
+
+In the past, entities used to be volatile.
+Home Assistant didn't track which entities it knew from the past and which it didn't.
+To allow customizations to entities, like renaming the entity or changing the unit of measurement, Home Assistant needed a way to keep track of each individual entity across restarts.
+
+To solve this, Home Assistant introduced the entity registry.
+The entity registry is a central place where Home Assistant keeps track of all entities it knows about.
+Each entity in the entity registry has a unique ID, which is unique per domain per platform.
+
+Ideally, the unique identifier is tied to the device itself.
+This way, if a user once need to remove and re-add an integration, the entity registry can automatically restore the previous entity customizations.
+It would also restore automations and dashboards that reference the entity.
```

Comment:
This make it sound like there's a backup restore of removed automations, but that's not what's happening. I think we just mean that existing automations will start working again when the entities, that the automations reference, are restored with the expected names and IDs.

Reply:
Honestly, I think we can just drop this whole line (the messaging will stay the same).

Reply:
I tweaked it a bit. I think I was also incorrect with that we restore all previous entity customizations